Source/Globals.h

```diff
@@ -24,7 +24,7 @@ enum class TextureType
 
 // Application -------------
 #define TITLE "Axolotl Engine"
-#define VERSION "1.0.0"
+#define VERSION "0.0.3"
```

Comment:
```suggestion
#define VERSION "0.3.0"
```
our last tag was that